Dockerfile

```diff
@@ -1,25 +1,24 @@
-FROM python:3.7.4-alpine3.10
+FROM amazonlinux
 
 RUN mkdir /cdk
 
-COPY ./cdk-packages.txt /cdk/
+COPY ./requirements.txt /cdk/
```

Comment:
this doesn't match the name of the file you updated. Ensure they are consistent.

Reply:
file name updated